testing/falkon/APKBUILD

```diff
@@ -0,0 +1,47 @@
+# Contributor: Leo <thinkabit.ukim@gmail.com>
+# Maintainer: Leo <thinkabit.ukim@gmail.com>
+pkgname=falkon
+pkgver=3.1.0
+pkgrel=0
+pkgdesc="Cross-platform Qt Web Browser"
+url="https://github.com/KDE/falkon"
+arch="all"
+license="GPL-3.0-or-later"
+depends="qt5-qtbase-sqlite"
+makedepends="cmake python3 qt5-qtbase-dev extra-cmake-modules qt5-qtdeclarative-dev
+ qt5-qtwebengine-dev xcb-util-dev libxcb-dev qt5-qtx11extras-dev qt5-qttools-dev
+ libgnome-keyring-dev"
+checkdepends="xvfb-run"
+subpackages="$pkgname-lang"
+source="https://download.kde.org/stable/falkon/3.1/falkon-${pkgver}.tar.xz
+	execinfo_h.patch"
+
+build() {
+	# Remove translations file, requires unpackaged KF5
+	rm -rf po
+
+	if [ "$CBUILD" != "$CHOST" ]; then
+		CMAKE_CROSSOPTS="-DCMAKE_SYSTEM_NAME=Linux -DCMAKE_HOST_SYSTEM_NAME=Linux"
+	fi
+	cmake \
+		-DCMAKE_INSTALL_PREFIX=/usr \
+		-DCMAKE_INSTALL_LIBDIR=lib \
+		-DBUILD_SHARED_LIBS=True \
+		-DCMAKE_BUILD_TYPE=Release \
+		-DCMAKE_CXX_FLAGS="$CXXFLAGS" \
+		-DCMAKE_C_FLAGS="$CFLAGS" \
+		-DBUILD_TESTING=ON \
+		${CMAKE_CROSSOPTS}
+	make
+}
+
+check() {
+	xvfb-run ctest -E '(falkon-webviewtest|falkon-webtabtest|falkon-tabmodeltest)'
```

Comment:
Might be nice to mention why they're disabled. In this case because they require working OpenGL.

Reply:
fixed